tests/dummy/app/templates/form.hbs

```diff
@@ -3,4 +3,33 @@
 <TestForm
   @form={{@model.form}}
   @formStore={{@model.formStore}}
-/>
\ No newline at end of file
+/>
+
+
+<AuHeading
+  @level={{1}}
+  @skin={{3}}
+>
+  Data
+</AuHeading>
+<AuTable
+  @size="large"
+>
+  <:title>Test</:title>
```

Comment:
I would remove the heading and change it to this, or remove the title block and only keep the heading.

```suggestion
  <:title>Data</:title>
```